pom.xml

```diff
@@ -255,11 +260,12 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <testcontainers.version>1.15.3</testcontainers.version>
-    <vertx.version>4.2.5</vertx.version>
```

Comment:
Is this version compatible with RMB? Should we update RMB to the version that depends on 4.2.6?

Reply:
Hi @okolawole-ebsco , we don't have RMB here in mod-inventory

Reply:
Great point!